src/pmns/stdpmid.pcp

```diff
@@ -158,7 +158,9 @@ QA_TEST_PERL	252
 SIMPLE		253
 ### FREE SLOT 254 ###
 MEMORY_PYTHON	255
-### MORE FREE SLOTS 256..510 ###
+### FREE SLOT 256 ###
+ROCESTAT	257
+### MORE FREE SLOTS 258..510 ###
 #
```

Comment:
Could you use 254 please?  It will simplify this part of the diff (you only need to change one line then).